recipe/meta.yaml

```diff
@@ -28,7 +28,7 @@ requirements:
     - libtool              # [not win]
     - make                 # [not win]
   host:
-    - boost-cpp
+    - libflint             # [not win]
```

Comment:
`libflint` is already listed below. Does this really not use Boost any more?

Reply:
Yes, they told me they don't.